.travis.yml

```diff
@@ -12,6 +15,12 @@ cache:
    - $PWD/public/lib/
    - $PWD/vendor/.bundle/
 
+before_install:
+  - yarn add mkdirp graceful-fs
+  - npm i -g npm@6.9.0
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.17.3
+  - export PATH="$HOME/.yarn/bin:$PATH"
+
```

Comment:
What is this supposed to be doing?

Reply:
I keep this as my yarn export path: 
```JS
export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
```

Reply:
@kaustubh-nair https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn